build-system/tasks/check-owners.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const deglob = require('globs-to-files');
+const log = require('fancy-log');
+const path = require('path');
+const {getOutput} = require('../common/exec');
+const {green, cyan, red, yellow} = require('ansi-colors');
+const {highlight} = require('cli-highlight');
+const {isTravisBuild} = require('../common/travis');
+
+const rootDir = path.dirname(path.dirname(__dirname));
+const prettierCmd =
+  'node_modules/.bin/prettier --parser json5 --config .prettierrc';
```

Comment:
Is `.prettierrc` needed here or would it default to the prettierrc file in the nearest directory?

Reply:
Good point. `--config .prettier` is implied, at which point `--parser json5` is superfluous. Removed both.